packages/server/src/modules/app/index.ts

```diff
@@ -56,3 +59,21 @@ application.start = () => {
   );
   socket.init(server);
 };
+
+if (process.env.LOCALHOST_CERTIFICATE_BASE_PATH) {
```

Comment:
More generic approach would be better
- Define two env variables instead one with path value

You don't want `LOCALHOST` specific behaviour or env variables. There is no reason server can't run on HTTPS in production.

This will affect you code bellow, something like

```
key:  fs.readFileSync(SSL_KEY_PATH),
cert: fs.readFileSync(SSL_CERT_PATH)
```